v3/lints/cabf_smime_br/lint_subscribers_crl_distribution_points_are_http.go

```diff
@@ -0,0 +1,75 @@
+/*
+ * ZLint Copyright 2023 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package cabf_smime_br
+
+import (
+	"net/url"
+
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+func init() {
+	lint.RegisterLint(&lint.Lint{
+		Name:          "e_subscribers_crl_distribution_points_are_http",
+		Description:   "cRLDistributionPoints SHALL have URL scheme HTTP.",
```

Comment:
Minor, but the BR uses 'URI' rather than 'URL' in this section.
```suggestion
		Description:   "cRLDistributionPoints SHALL have URI scheme HTTP.",
```

Reply:
I suggest a similar change to the details of the error if you opt to make this change. Otherwise the PR looks good to me.

Reply:
Done. Thanks for the review.